src/domain/common/whiteboard-rt/dto/whiteboard.rt.dto.create.ts

```diff
@@ -0,0 +1,21 @@
+import { Field, InputType } from '@nestjs/graphql';
+import { IsOptional, MaxLength } from 'class-validator';
+import { CANVAS_VALUE_LENGTH } from '@common/constants';
+import { NameID } from '@domain/common/scalars/scalar.nameid';
+import { CreateNameableInput } from '@domain/common/entity/nameable-entity/dto/nameable.dto.create';
+
+@InputType()
+export class CreateWhiteboardRtInput extends CreateNameableInput {
+  @Field(() => String, { nullable: true })
+  @IsOptional()
+  @MaxLength(CANVAS_VALUE_LENGTH)
+  value?: string;
```

Comment:
please rename this field to be 'content', I will shortly be putting up a PR for making this change also on the Whiteboard entity. 'value' is just a terrible name...